src/lib/layouts/workflow-run-layout.svelte

```diff
@@ -133,32 +144,45 @@
   });
 </script>
 
-<LabsModeGuard>
-  <div
-    class="absolute bottom-0 left-0 right-0 {$viewDataEncoderSettings
-      ? 'top-[540px]'
-      : 'top-0'}
+{#if raw}
+  <div class="relative h-auto whitespace-break-spaces rounded bg-primary p-4">
```

Comment:
Might want to add `overflow-x` 
```suggestion
  <div
    class="relative h-auto overflow-x-auto whitespace-break-spaces rounded bg-primary p-4"
  >
```
or `break-words`
```suggestion
  <div
    class="relative h-auto whitespace-break-spaces break-words rounded bg-primary p-4"
  >
```
for small screens ⬇️ 

![Screenshot 2024-06-27 at 4 16 36 PM](https://github.com/temporalio/ui/assets/15069288/3ad51ea2-7ffd-4df5-b8cb-1e374193af95)

Reply:
Good call